charts/gardener/controlplane/charts/runtime/templates/apiserver/createNewCA.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.global.apiserver.createNewCa }}
```

Comment:
While I do support the idea for having more flexibility of using an external CA instead of the inline one, I don't think the core Gardener repo is a good place for templates that render `cert-manager.io/v1` resources. This template hardly depends on anything in Gardener, and Gardener has nothing to do with `cert-manager `currently. I think it would be more appropriate to put such charts in a different repo and render them as part of the overall Gardener setup process, rather than having the core Gardener render them.

This basically means that the entire `createNewCa` option should be removed. It is already handled practically identically to `caFromSecret` anyway, except for this template.

In case the template stays after all, it should be renamed to `cert-manager-ca.yaml` or something similar, the current name does not match our conventions.